client/test/test_helper.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import TestUtils from 'react-addons-test-utils';
+import jsdom from 'jsdom';
+import chai from 'chai';
+import chaiImmutable from 'chai-immutable';
+
+const doc = jsdom.jsdom('<!doctype html><html><body></body></html>');
+const win = doc.defaultView;
+
+global.document = doc;
+global.window = win;
+
+// from mocha-jsdom https://github.com/rstacruz/mocha-jsdom/blob/master/index.js#L80
+function propagateToGlobal(window) {
+  for (const key in window) {
+    if (!window.hasOwnProperty(key)) continue;
```

Comment:
this is OK. I'm perfectly happy if you want to use lodash and make this a bit more elegant. 

https://lodash.com/docs#forOwn

I'm of the opinion of using lodash more in this demo app.